mono/utils/mono-codeman.c

```diff
@@ -137,6 +145,9 @@ free_chunklist (CodeChunk *chunk)
 	for (; chunk; ) {
 		dead = chunk;
 		mono_profiler_code_chunk_destroy ((gpointer) dead->data);
+#if MONO_ARCH_HAVE_UNWIND_TABLE
+		RtlDeleteFunctionTable((DWORD64)dead->data | 0x03);
```

Comment:
Can you get the return code and warn/assert? I am afraid to use `g_assert` since that is enabled even in release. Maybe put behind a `_DEBUG` doing `g_critical/g_warning` in release and `g_error` in debug?